test/libp2p/crypto/key_generator_test.cpp

```diff
@@ -108,3 +111,18 @@ TEST_P(KeyLengthTest, KeyLengthCorrect) {
   EXPECT_OUTCOME_TRUE_2(val, keygen_.generateKeys(key_type))
   ASSERT_EQ(val.privateKey.data.size(), private_key_length);
 }
+
+// TODO(turuslan): convert to TestWithParam and test more key types
```

Comment:
Our policy regarding TODOs is "every todo is backed by a task in Jira".
Thus, please either create a task or omit the todo.

In my view, there is no need for a parametrized test case now. On the other hand, that might be required when more generation/derivation algorithms will be tested.